main.go

```diff
@@ -35,16 +35,21 @@ func handleCommand(command string) {
 	case "cart":
 		fmt.Println("Your cart is currently empty.")
 	case "quit":
-		break
+		fmt.Println("(exit the program with code 0)")
```

Comment:
Okay, well, I understand that this part of the original requirements document may have been a little bit ambiguous, but you don't need to print this text out, just exit the program. Or, you can print a cute message like "Bye~"
The easiest way to stop the process at any point is using [os.Exit](https://pkg.go.dev/os#Exit), it just quits the program without any other changes to the program's structure.